app/controllers/api_controller/error_handler.rb

```diff
@@ -35,7 +35,10 @@ def api_error(kind, message, klass, backtrace, status)
       # We don't want to return the stack trace, but only log it in case of an internal error
       api_log_error("\n\n#{backtrace}") if kind == :internal_server_error && !backtrace.empty?
 
-      render :json => {:error => err}, :status => status
+      json = {:error => err}
+      json[:backtrace] = backtrace if Rails.env.test?
+      render :json => json, :status => status
+
```

Comment:
I don't mind, but I do prefer to have that as a formal element in the error hash itself. i.e. just add a single
line after 33 above as follows:

```
err[:backtrace] = backtrace if !backtrace.empty && Rails.env.test?
```

Do we want this on 400's too or just 500 ?